### PR TITLE
Closes #275 - Useful error message for invalid input parameter

### LINKED ISSUE #275: Improve error message when input file doesn't exist in 'gtfs2ntfs'
Bonjour,

J'ai vu qu'il y avait eu un PR (PR 187) en février ajoutant ce message d'erreur mais je n'arrive pas à récupérer plus d'informations...

Du coup je lance ma commande : 
`target/release/gtfs2ntfs -i /gtfs/gtfs-test.zip -o /ntfs/ntfs-test.zip`

Et j'ai directement un retour :
`calendar_dates.txt or calendar.txt not found`

Or mon GTFS contient bien un calendar.txt (j'ai testé sur plusieurs GTFS avec tous une origine différente) et ce sont des GTFS qui s'intègrent bien dans mon navitia...

Auriez-vous une idée du soucis ? Pour information je suis sur Proj4 v6.1.0

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: src/bin/gtfs2ntfs.rs [hunk 1]
```diff
@@ -65,8 +66,10 @@ fn run() -> Result<()> {
         transit_model::gtfs::read_from_url(&url, opt.config_path, opt.prefix)?
     } else if opt.input.is_file() {
         transit_model::gtfs::read_from_zip(opt.input, opt.config_path, opt.prefix)?
-    } else {
+    } else if opt.input.is_dir() {
         transit_model::gtfs::read_from_path(opt.input, opt.config_path, opt.prefix)?
+    } else {
+        bail!("Invalid input data: must be an existing directory or a ZIP archive");
```
Comment (datanel) [inline src/bin/gtfs2ntfs.rs]: or an url

**Comment by datanel** [comment]
you have to rebase

**Comment by woshilapin** [comment]
I just rebased this PR.

File: src/bin/gtfs2ntfs.rs (hunk 1 above)
Comment (woshilapin) [reply src/bin/gtfs2ntfs.rs]: Fix for `gtfs2ntfs` and `kv12ntfs` in 0548c88614d80ab3711833a25529ac9fee089d97.

**Pull request merged**